src/explore-education-statistics-common/src/services/permalinkService.ts

```diff
@@ -4,6 +4,7 @@ import { ConfiguredTable } from '@common/services/types/table';
 import deduplicatePermalinkLocations from '@common/services/util/permalinkServiceUtils';
 
 export type Permalink = ConfiguredTable & {
+  created: string;
```

Comment:
On the BE `FastTrackViewModel` had a created property but it was the created data of the data block and I couldn't see what relevance it had so I removed it.

In terms of the FE code, this change was necessary because in `fastTrackService.ts` and `permalinkService.ts` we had both:

```
export type FastTrackTable = ConfiguredTable & { ... }
export type Permalink = ConfiguredTable & { ... }
```

and `created` was on `ConfiguredTable`.

So now it's been removed from `ConfiguredTable` in the change made to `table.ts` and been added here to `Permalink`.